### PR TITLE
Improve GitHub Sponsors UX

### PR DESCRIPTION
**Pull request by fregante**
Just a silly feel-good feature.

gif

Named `rgh-` because it's safe so it doesn't need to be documented, but it's not limited to RG.

I'll probably also add an animation on [Sponsor] click

## Test

https://github.com/sindresorhus/refined-github/issues
https://github.com/babel

### CODE REVIEW

**Comment by fregante** [comment]
gif

File: source/features/rgh-sponsor-button.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+void features.add(__filebasename, {
+	include: [
+		pageDetect.isIssue,
+		pageDetect.isRepoIssueList
+	],
+	init: handleNewIssue
+});
+
+void features.add(__filebasename, {
```
Comment (yakov116) [inline source/features/rgh-sponsor-button.tsx:73]: Why the second features.add?

Comment (fregante) [reply source/features/rgh-sponsor-button.tsx:73]: D’oh. I forgot we merged them 😂

**Comment by yakov116** [comment]
Not by a computer... Does this check if they are sponsoring already?

**Comment by fregante** [comment]
I don’t think that piece of information is available on the page, right? The button appears as “Sponsoring” on profiles but not on repos

**Review by 641210 — COMMENTED** [review]
****

**Pull request merged**